playbooks/openstack/configuration.md

```diff
@@ -778,6 +780,45 @@ enabled_handlers = vif,lb,lbaasspec,namespace,*kuryrnet*
 Note depending on the Kuryr controller image version, this feature (handler)
 may not be available.
 
+The `kuryr_openstack_global_namespace` defines the list of namespaces that are
+global and can be accessed from any other namespace, as well as access any
+namespace. By default only the `default` and the `openshift-monitoring`
+namespaces are included in the list.
+
+Note the access from the other namespaces to the global ones is handled by
+the Security Group named `*-allow_from_default`. Although this can be handle
+with the default `remote_group_id` rule existing there, this has proven to cause
+scaling issues and some times even connectivity issues. To avoid those the next
+actions are highly recomended:
+
+```
+# Remove the security group rule with the remote_group_id
+$ openstack security group show *-allow_from_default | remote_group_id
+$ openstack security group rule delete REMOTE_GROUP_ID
+
+# Add a rule per each namespace added into the kuryr_openstack_global_namespace
```

Comment:
Shouldn't those steps be reversed? I'd rather add rules first, then remove the remote_group_id one to avoid disruption.